packages/x-components/jest.config.js

```diff
@@ -1,5 +1,9 @@
 module.exports = {
-  preset: '@vue/cli-plugin-unit-jest/presets/typescript',
```

Comment:
I removed this dependency and now we're using jest, as the @vue/cli-plugin-unit-jest uses the v24 of jest.